src/DynamoCoreWpf/ViewModels/Search/BrowserItemViewModel.cs

```diff
@@ -430,9 +430,12 @@ private void RemoveFromItems(IEnumerable<ISearchEntryViewModel> oldItems)
 
```

Comment:
Can you help to see if `RecursivelySort` can be removed (it's not used anywhere), because it replaces `Items` without subscribing to its `Items.CollectionChanged` event.